Cargo.toml

```diff
@@ -18,6 +18,9 @@ members = [
     "examples/qml_features/rust",
     "examples/qml_minimal/rust",
     "examples/qml_basics",
+    "examples/qml_multi_crates/rust/main",
+    "examples/qml_multi_crates/rust/sub1",
+    "examples/qml_multi_crates/rust/sub2",
```

Comment:
should you add a changelog entry stating that this is now possible? (multi crate import/qml modules etc)

Reply:
Done